### ISSUE TITLE
Improve build speed with runtime opts

### ISSUE THREAD

**Issue by andys8** [halfzebra/create-elm-app#321]
# TL;DR

Build time can potentially be improved by passing `runtimeOptions` / RTS opts.

# Issue

The Elm 0.19 is fast. People are still experiencing additional performance improvements by passing RTS options, which is now possible without recompiling the compiler.

# Flags

"Antew" wrote this
> We saw a 40% improvement in build times with `+RTS -A128M -H128M -n8m -RTS`, though 40% improvement is a lot less dramatic in 0.19!

https://elmlang.slack.com/archives/C13L7S5GR/p1542423683071500?thread_ts=1542398135.069600&cid=C13L7S5GR

More discussion on elm slack #compile-time.

# Changes

`elm-webpack-loader` supports passing `runtimeOptions`.
https://github.com/elm-community/elm-webpack-loader/tree/25e7588dfbb789f6d30de91f0bd395aed1e04db7#runtimeoptions-default-undefined

**Comment by halfzebra** [comment]
Hi @andys8!

Thanks for pointing that out!

Would you be interested in working on a PR for this?

**Comment by antew** [comment]
One way to use it now is through the `GHCRTS` environment var, you can use that to set the config as well, e.g. `export GHCRTS="-A128M -H128M -n8m"`

**Comment by halfzebra** [comment]
I tend to agree, but it was worth checking it out 🙂

**Comment by halfzebra** [comment]
Thanks for participating in this little experiment!

I'm closing the issue due to the fact that the additional runtime configuration does not have a significant impact on the compilation.

Please feel free to reopen this issue if you feel like it's necessary!

**Comment by andys8** [comment]
The changes are in `master` and need to be reverted.

**Comment by halfzebra** [comment]
Thanks for remembering, that's takien care of already :slightly_smiling_face:
